--- application.py	2021-07-08 01:16:30.000000000 +0000
+++ application.py	2021-07-08 01:42:28.244735409 +0000
@@ -63,6 +63,8 @@
     template_folder=str(resource_path("templates")),
     static_folder=str(STATIC_FOLDER),
 )
+from werkzeug.middleware.proxy_fix import ProxyFix
+app.wsgi_app = ProxyFix(app.wsgi_app, x_for=1, x_proto=1, x_host=1, x_port=1, x_prefix=0)
 app.register_blueprint(json_api, url_prefix="/<bfile>/api")
 
 app.json_encoder = FavaJSONEncoder  # type: ignore
